alacritty/src/event.rs

```diff
@@ -460,10 +460,13 @@ impl<N: Notify + OnResize> Processor<N> {
         match event {
             GlutinEvent::UserEvent(event) => match event {
                 Event::DPRChanged(scale_factor, (width, height)) => {
-                    let display_update_pending = &mut processor.ctx.display_update_pending;
+                    let display_update_pending = &mut ctx.display_update_pending;
```

Comment:
processor.ctx

Reply:
Oops, tested out to extract that and looks like I've missed this when undoing. Thanks.